PaddleCV/PaddleDetection/ppdet/core/config/schema.py

```diff
@@ -160,6 +161,26 @@ def validate(self):
                 self.name, ", ".join(mismatch_keys)))
 
 
+class SharedConfig(object):
+    r"""A specialized `inject` type for simple values
```

Comment:
Make the doc comments unified with others

Reply:
done